app/models/review.rb

```diff
@@ -0,0 +1,24 @@
+# == Schema Information
+#
+# Table name: reviews
+#
+#  id          :integer          not null, primary key
+#  reviewer_id :integer          not null
+#  reviewed_id :integer          not null
+#  title       :string           not null
+#  body        :text             not null
+#  rating      :integer          not null
+#  project_id  :integer          not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+
+class Review < ActiveRecord::Base
+  validates :reviewer_id, :reviewed_id, :project_id, :rating, :title, :body, presence: true
+  validates :title, :body, length: { minimum: 2 }
+  validates :rating, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 100 }
```

Comment:
I don't think ratings need to be their own separate model, we will have a form for leaving a review right? The review's average rating generates the rating number?